actionpack/lib/action_controller/metal/rate_limiting.rb

```diff
@@ -26,38 +30,24 @@ module ClassMethods
       #       by: -> { request.domain }, with: -> { redirect_to busy_controller_url, alert: "Too many signups on domain!" }, only: :new
       #   end
       #
-      # Note: Rate limiting relies on the application having an accessible Redis server and on Kredis 1.7.0+ being available in the bundle.
-      # This uses the Kredis limiter type underneath, which is failsafe, so in case Redis is inaccessible, the rate limit will not refuse action execution.
-      def rate_limit(to:, within:, by: -> { request.remote_ip }, with: -> { head :too_many_requests }, **options)
-        require_compatible_kredis
-        before_action -> { rate_limiting(to: to, within: within, by: by, with: with) }, **options
+      #   class APIController < ApplicationController
+      #     RATE_LIMIT_STORE = ActiveSupport::Cache::RedisCacheStore.new(url: ENV["REDIS_URL"])
+      #     rate_limit to: 10, within: 3.minutes, store: RATE_LIMIT_STORE
+      #   end
+      #
+      # TODO Note
```

Comment:
Noted! :heavy_check_mark:

:question:

Reply:
🤦 

Thanks, fixed in 10a34b7621a99fc9c126d43b42eda7e31b138278